06-improve-agent-performance/improvement-overview.ipynb

```diff
@@ -19,9 +19,13 @@
         "- Evaluation Job\n",
```

Comment:
<p>remove output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ContextualAI/examples/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>yes, removing all output from the notebook, need to get a checklist</p>